styles.css

```
@@ -0,0 +1,249 @@
+        4.4 Services
+        4.5 Form
+        4.6 Footer
+    5. Media Queries
+////////////////////////*/
+
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+/* Paars #5722CD;
+Blauw #4845E4;
+Roze #ED4385;
+Zwart #0C0C0C (voor standaard tekst);
+Lichtpaars #F2F0FF (voor de grote zwevende woorden, zoals "hello" en "services") Uitzonderingen:
+Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)
+Grijs #7A7A7A (voor de service-tegel tekst);*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', 'Arial', sans-serif;
+    font-style: normal;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-family: 'Merriweather', 'Times New Roman', serif;
+    font-style: normal;
+}
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+/* --------------- 3.1 Reusable ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+header {
+    background: linear-gradient(90deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 100%);
+    position: relative;
+}
+
+.navigation-bar{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.navigation-list {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 350px;
+    padding: 20px;
+}
+
+.jump-link-nav {
+    color: #FFFFFF;
+    font-style: normal;
+    text-decoration: none;
+}
+
+.introduction {
+    color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 100px;
+    margin: 0 200px 100px 200px;
+}
+
+.introduction-header {
+    font-size: 50px;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.introduction-line {
+    font-size: 35px;
+    padding: 20px;
+    margin: 20px;
+}
+
+.introduction-image {
+    width: 100px;
+    height: 80px;
+    bottom: 0;
+    position: absolute;
+    left: calc(50% - 50px);
+    margin-top: 60px;
+}
+
+.work-together-button {
+    background-color: #ED4385;
+    font-size: 15px;
+    color: #FFFFFF;
+    text-transform: uppercase;
+    border: none;
+    border-radius: 50px;
+    padding: 20px;
+    margin: 20px;
+    width: 250px;
+    box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.15);
+}
+
+/* --------------- 4.2 Hello Column ------------------- */
+
+.hello {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: 0 200px 0 200px;
```

Hier kun je ook dit doen
```suggestion
    margin: 0 200px;
```